app/views/logs/delete_logs.html.erb

```diff
@@ -0,0 +1,61 @@
+<% title = "Delete logs" %>
+<% content_for :title, title %>
+<% content_for :before_content do %>
+  <%= govuk_back_link(href: :back) %>
+<% end %>
+
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= title %></span>
+  Review the logs you want to delete
+</h1>
+<% logs = @delete_logs_form.logs %>
+<p>You've selected <%= logs&.count %> <%= "log".pluralize(logs&.count) %> to delete</p>
+
+<%= form_with model: @delete_logs_form, url: delete_logs_confirmation_lettings_logs_path, method: "get" do |f| %>
+  <%= f.hidden_field :log_type, value: @delete_logs_form.log_type %>
+  <%= f.hidden_field :search_term, value: @search_term %>
+  <%= f.govuk_error_summary %>
+  <%= govuk_table do |table| %>
+    <% table.head do |head| %>
+      <% head.row do |row| %>
+        <% row.cell(header: true, text: "Log ID") %>
+        <% case @delete_logs_form.log_type %>
+        <% when :lettings %>
+          <% row.cell(header: true, text: "Tenancy code") %>
+          <% row.cell(header: true, text: "Property reference") %>
+        <% when :sales %>
+          <%# for sales display owning org name and creation date %>
+        <% end %>
+        <% row.cell(header: true, text: "Status") %>
+        <% row.cell(header: true, text: "Delete?") %>
+      <% end %>
+    <% end %>
+    <% table.body do |body| %>
+      <% f.govuk_check_boxes_fieldset :logs_to_delete, small: true do %>
+        <% logs.each do |log| %>
+          <% body.row do |row| %>
+            <% row.cell(text: log.id) %>
+            <% case @delete_logs_form.log_type %>
+            <% when :lettings %>
+              <% row.cell(text: log.tenancycode) %>
+              <% row.cell(text: log.propcode) %>
+            <% when :sales %>
+              <%# for sales display owning org name and creation date %>
+            <% end %>
+            <% row.cell(text: status_tag(log.status)) %>
+            <% row.cell do %>
+              <% f.check_box :bla %>
+              <% f.govuk_check_box :logs_to_delete, log.id,
```

Comment:
```suggestion
              <% f.govuk_check_box :log_ids, log.id,
```

Reply:
there is already a variable log_ids representing all the logs in the table.
Do you want the change so that it is expressing that this thing is a collection of ids rather than actual logs?
I think here in the context of this form and consistent with the naming in the controller methods I will change this to selected_ids unless you don't like that